packages/easy-email-extensions/src/components/Form/RichTextToolBar/components/Tools/Tools.tsx

```diff
@@ -109,6 +126,157 @@ export function Tools(props: ToolsProps) {
 
   const getPopoverMountNode = () => document.getElementById(FIXED_CONTAINER_ID)!;
 
+  const enabledTools = toolbar?.tools ?? Object.values(AvailableTools);
+
```

Comment:
Using Object.values ​​will depend on the order of AvailableTools keys. It is recommended to enumerate all the current Available Tools.
```suggestion

  const enabledTools = toolbar?.tools ??  [AvailableTools.MergeTags, AvailableTools.FontFamily, ...];

```